changelog/split-std-datetime.dd

```diff
@@ -0,0 +1,43 @@
+std.datetime has been split into a package.
+
+std.datetime is now a package containing the following modules:
+
+$(UL
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_common.html">std.datetime.common</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_date.html">std.datetime.date</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_datetime.html">std.datetime.datetime</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_interval.html">std.datetime.interval</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_systime.html">std.datetime.systime</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_timeofday.html">std.datetime.timeofday</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_timezone.html">std.datetime.timezone</a>)
+)
+
+$(LI <a href="$(PHOBOS_PATH)std_datetime.html">std.datetime.package</a>)
```

Comment:
This shouldn't block this PR as we always can make a short follow-up.
Anyhow in case you are interested, there's a short summary in the [README](https://github.com/dlang/phobos/tree/master/changelog).

Other examples:
- https://github.com/dlang/phobos/tree/v2.074.0-rc1/changelog
- https://github.com/dlang/phobos/blob/v2.074.0-rc1/changelog/std-experimental-checkedint.dd
- https://github.com/dlang/phobos/blob/v2.074.0-rc1/changelog/std-format-formattedRead.dd